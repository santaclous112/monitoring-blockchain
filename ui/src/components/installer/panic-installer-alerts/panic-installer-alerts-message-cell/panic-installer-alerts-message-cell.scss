.panic-installer-alerts-message-cell {
  &__container {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    padding-top: 20px;

    svc-label {
      font-weight: 900;
      margin-left: 3px;
    }

    svc-text {
      font-size: 0.8em !important;
      margin-left: 10px;
    }
  }
}