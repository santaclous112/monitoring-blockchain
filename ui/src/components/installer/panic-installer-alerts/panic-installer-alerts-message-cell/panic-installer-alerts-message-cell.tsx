import {Component, h, Prop} from '@stencil/core';

@Component({
  tag: 'panic-installer-alerts-message-cell',
  styleUrl: 'panic-installer-alerts-message-cell.scss'
})
export class PanicInstallerAlertsMessageCell {

  /**
   * the label which serves as the header text
   */
  @Prop() label: string;
  /**
   * the message which serves as additional text
   */
  @Prop() message: string;

  render(){
    return (
      <div class={"panic-installer-alerts-message-cell__container"}>
        <svc-label>{this.label}</svc-label>
        <svc-text>
          <p>
            {this.message}
          </p>
        </svc-text>
      </div>
    )
  }
}
