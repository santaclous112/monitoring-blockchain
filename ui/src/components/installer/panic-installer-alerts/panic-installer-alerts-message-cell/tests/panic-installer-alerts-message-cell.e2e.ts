import { newSpecPage } from '@stencil/core/testing';
import {PanicInstallerAlertsMessageCell} from "../panic-installer-alerts-message-cell";

describe('panic-installer-alerts-message-cell', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [PanicInstallerAlertsMessageCell],
      html: '<panic-installer-alerts-message-cell label="test" message="test">' +
        '</panic-installer-alerts-message-cell>',
    });

    const panicInstallerAlertsMessageCell = page.body.querySelector('panic-installer-alerts-message-cell');
    expect(panicInstallerAlertsMessageCell).toMatchSnapshot();
  });
});