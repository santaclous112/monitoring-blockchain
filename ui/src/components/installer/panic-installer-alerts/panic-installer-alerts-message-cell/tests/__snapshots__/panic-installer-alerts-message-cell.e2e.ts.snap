// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-installer-alerts-message-cell renders 1`] = `
<panic-installer-alerts-message-cell label="test" message="test">
  <div class="panic-installer-alerts-message-cell__container">
    <svc-label>
      test
    </svc-label>
    <svc-text>
      <p>
        test
      </p>
    </svc-text>
  </div>
</panic-installer-alerts-message-cell>
`;
